CONTRIBUTING.md

```diff
@@ -77,7 +77,7 @@ Here's a list of the big ones:
 * [status-bar](https://github.com/atom/status-bar) - the status bar at the bottom of the UI.
 * [markdown-preview](https://github.com/atom/markdown-preview) - the rendered markdown pane item.
 * [settings-view](https://github.com/atom/settings-view) - the settings UI pane item.
-* [autocomplete-plus](https://github.com/atom/autocomplete-plus) - autocompletions shown while typing. Some languages have additional packages for autocompletion functionality, such as [autocomplete-html](https://github.com/atom/autocomplete-html).
+* [autocomplete-plus](https://github.com/atom/autocomplete-plus) - auto-completions shown while typing. Some languages have additional packages for autocompletion functionality, such as [autocomplete-html](https://github.com/atom/autocomplete-html).
```

Comment:
This is an unnecessary change, especially when the name of the package is `autocomplete` without a hyphen.